packages/release-please/test/release-please.ts

```diff
@@ -263,6 +264,51 @@ describe('ReleasePleaseBot', () => {
       requests.done();
       assert(executed, 'should have executed the runner');
     });
+
+    describe('for manifest releases', () => {
+      it('should build a release PR', async () => {
+        let executed = false;
+        Runner.manifest = async (manifest: Manifest) => {
+          executed = true;
+        };
+        const config = fs.readFileSync(
+          resolve(fixturesPath, 'config', 'manifest.yml')
+        );
+        const requests = nock('https://api.github.com')
+          .get(
+            '/repos/chingor13/google-auth-library-java/contents/.github%2Frelease-please.yml'
+          )
+          .reply(200, config);
+
+        await probot.receive({name: 'push', payload, id: 'abc123'});
+        requests.done();
+        assert(executed, 'should have executed the runner');
+      });
+
+      it('should handle GitHub releases, if configured', async () => {
+        let runnerExecuted = false;
+        let releaserExecuted = false;
+        Runner.manifest = async (manifest: Manifest) => {
```

Comment:
You could use sinon and replace this with something like:

```
const stub = sinon.stub(Runner, 'manifest');
sinon.assert.calledOnce(stub)
```

That way we get rid of the complained about `Manifest` being unused.

Reply:
Will do in a follow-up PR.